src/components/Library/LibraryBook/DoneEl/DoneEl.tsx

```diff
@@ -36,7 +36,7 @@ const DoneEl: FC<Props> = ({ _id, rating, resume, onUpdate }) => {
             <ModalResume
               bookId={_id}
               rating={rating}
-              resume={resume}
+              resume={resume ? resume : ''}
```

Comment:
Please, cancel it. On the backend is only string type/ I will change it  in Trello.

Reply:
Fixed